about.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8"/>
+  <meta name="viewport" content="width=device-width, initial-1">
+  <meta name="description" content="IJS Summit 2022 about page">
+  <meta name="Author" content="Hemerson Foreste">
+  <link rel="stylesheet" type="text/css" href="style.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css"/>
+  <title>CC Global Summit 2015 about page</title>
+</head>
+<body> 
+    <main class="main-about">   
+      <header>
+        <div class="top-header">
+            <ul class="top-header-links">
+              <li><a class="facebook" href="" title="facebook"><i class="fab fa-facebook-f"></i></a></li>
+              <li><a class="twitter" href="" title="twitter"><i class="fab fa-twitter"></i></a></li>
+              <li>English</li>
+              <li>My Page</li>
+              <li>Logout</li>
+            </ul>
+        </div>        
+        <div class="btn-hamburger-menu mobile-version"><i class="fas fa-bars"></i></div>        
+        <div class="btn-close mobile-version"><i class="fas fa-times"></i></div> 
+        <nav class="default-navbar">  
+          <div class="logo-container">
+            <a href="index.html" class="home logo" title="home"><img src="images/LogoHemer.png" alt="logo"></a>
+          </div>                  
+          <ul class="menu">
```

Comment:
- [ ] **Required** In the mobile version, After clicking on bars, there should be a `close button` to close the menu bar.
![jlk](https://user-images.githubusercontent.com/26767383/152565154-3848fb55-6659-485f-925b-4ecf3df50b56.PNG)